### PR TITLE
add note about defer toggle not avail for sl commands

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr temporarily adds a quick note letting users know that the defer toggle isn't enabled for sl/mf commands, however this will soon change.

Also adding since there are sl-related workshops this week during coalesce

[internal slack thread](https://dbt-labs.slack.com/archives/C04REFN03MX/p1728043685018029)

### CODE REVIEW

**Review by matthewshaver — APPROVED** [review]

File: website/docs/docs/build/metricflow-commands.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ dbt Cloud jobs support the `dbt sl validate` command to [automatically test your
 In dbt Cloud, run MetricFlow commands directly in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). 
 
 For dbt Cloud CLI users, MetricFlow commands are embedded in the dbt Cloud CLI, which means you can immediately run them once you install the dbt Cloud CLI and don't need to install MetricFlow separately. You don't need to manage versioning because your dbt Cloud account will automatically manage the versioning for you.
+
+<!--remove when fixed -->
+Note: The **Defer to staging/production** [toggle](/docs/cloud/about-cloud-develop-defer#defer-in-the-dbt-cloud-ide) is currently not available when running Semantic Layer commands in the dbt Cloud IDE.  To use defer for Semantic layer commands in the IDE, manually add the `--defer` flag to the command. This is a temporary workaround and will be available soon.
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md:32]: Should note that you need to have the toggle on as well AND you need the --defer

**Review by WilliamDee — APPROVED** [review]

File: website/docs/docs/build/metricflow-commands.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/build/metricflow-commands.md:32]: oh that's a good shout! so users need to have the toggle on and then use the --defer flag @WilliamDee ?

Comment (WilliamDee) [reply website/docs/docs/build/metricflow-commands.md:32]: Thats correct

Comment (mirnawong1) [reply website/docs/docs/build/metricflow-commands.md:32]: ok good to know, i'll update it!

Comment (mirnawong1) [reply website/docs/docs/build/metricflow-commands.md:32]: thanks @WilliamDee ! i've updated it in this pr here https://github.com/dbt-labs/docs.getdbt.com/pull/6255

**Pull request merged**